package.json

```diff
@@ -20,7 +20,8 @@
   "license": "ISC",
   "homepage": "https://github.com/leoasis/react-sound",
   "dependencies": {
-    "react": ">=0.13.3 <16.0.0",
+    "prop-types": "15.5.8",
```

Comment:
Whoops yeah, forgot to change my npm config, (on a related note, what are your thoughts on supporting yarn in this project, (just adding the lockfile that is)?)